samples/wifi/scan/prj.conf

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+
+# System settings
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_NANO=n
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_LOG=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=y
+CONFIG_NET_DHCPV4=y
+CONFIG_DNS_RESOLVER=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=153600
```

Comment:
can reduce as its only scan.

Reply:
> can reduce as its only scan.

@krish2718 Reduced to 100000, seeing below errors.
```
<inf> wifi_nrf: wifi_nrf_fmac_fw_load: UMAC boot check passed
<err> wifi_nrf: wifi_nrf_fmac_rx_cmd_send: No space for allocating RX buffer
```

Reply:
okay, lets leave this as is for now, my memory optimization changes will take care of this.